binscripts/rvm-installer

```diff
@@ -27,7 +27,9 @@ rvm_install_initialize()
 
 log()  { printf "%b\n" "$*"; }
 debug(){ [[ ${rvm_debug_flag:-0} -eq 0 ]] || printf "%b\n" "$*" >&2; }
+notice() { log "$*" >&2 ; }
 fail() { log "\nERROR: $*\n" >&2 ; exit 1 ; }
```

Comment:
This should probably change to `fail_with_code 1 "$*"`, right?

Reply:
Good call sir, fixed